src/schema/schema_triggers.py

```diff
@@ -2513,11 +2513,14 @@ def update_status(property_key, normalized_type, user_token, existing_data_dict,
     new_data_dict : dict
         A merged dictionary that contains all possible input data to be used
     """
+    status = existing_data_dict.get('status')
+
     # execute set_status_history
     set_status_history(property_key, normalized_type, user_token, existing_data_dict, new_data_dict)
 
-    # execute sync_component_dataset_status
-    sync_component_dataset_status(property_key, normalized_type, user_token, existing_data_dict, new_data_dict)
+    if status is not None and not equals(status, 'published'):
```

Comment:
@libpitt I believe it would be `Published`. You can also check against `status.lower()`